### DISCUSSION TITLE
Metrics to evaluate state of dagrun

### DISCUSSION THREAD

**Discussion by kangtao** [apache/airflow#26738] [Q&A]
Hi,

By looking at https://airflow.apache.org/docs/apache-airflow/stable/logging-monitoring/metrics.html, I see I can use ti.finish.<dag_id>.<task_id>.<state> to tell the final state of specific task id from specific dag id. But what if I do not need task id? I want a metric to say how many dagruns are succeeded or failed for any specific dag id I want in certain time range. Does airflow have a metrics like that?

Thanks

**Comment by potiuk** [comment]
Those are all metrics you see in the docs. If you cannot use those and need new one - you can make a PR and contribute it, and add one (and apply a patch locally even if you want to use it).

**Reply by kangtao** [reply]
@potiuk Thanks. I notice that I can probably use the count of dagrun.duration.success and dagrun.duration.failed to get the dag run metrics I want.
